src/web/environments/environment.ts

```diff
@@ -1,4 +1,4 @@
-import { config } from './config';
+import { config } from './config.template';
```

Comment:
same here

Reply:
The link was broken and tests were not passing so I updated the link path and its fine now. But sure I will remove it if you will persist.

Reply:
@SimonHasak template files are not meant to be used directly. If you have followed the step mentioned in https://github.com/TEAMMATES/teammates/blob/master/docs/setting-up.md#step-3-set-up-project-specific-settings-and-dependencies, you should have a `config.ts` file.

Reply:
Yes I see now. Thanks for explanation ;).